### ISSUE TITLE
Concrete segment lists of Panoramic Video Panoptic Segmentation dataset

### ISSUE THREAD

**Issue by Nole21** [waymo-research/waymo-open-dataset#888]
I wonder the concrete segment lists of the validation and test set of PVP dataset in v1.4.1, since not all segments in the Waymo Open Data is labeled.

**Comment by Nole21** [comment]
@alexzzhu Can you kindly offer me some clues?

**Comment by alexzzhu** [comment]
Does the list in tutorial/2d_pvps_training_frames.txt work?

**Comment by Nole21** [comment]
Thank you for your quick reply. I will try later.

**Comment by Nole21** [comment]
Is segmentation sequence in test and validataion longer than that in training set? @alexzzhu

**Comment by Nole21** [comment]
The segmentation in training set is not global as you mentioned in the paper, you divide a whole sequence into 4 five-frame sequences. And I found the instance segmentation id changed in different sub-sequences(even after mapping instance id to global). Can you offer some long-sequence segment lists?
![image](https://github.com/user-attachments/assets/fe91434f-6a30-48f6-9ad7-fbb07d6e47a8)

**Comment by alexzzhu** [comment]
Yes this is correct, training is split into five frame sequences as described in the paper. The only longer sequences available are in the validation set.
